frontend/app/components/person-new.js

```diff
@@ -99,7 +100,7 @@ export default ApplicationComponent.extend({
               `person.${attribute}`
             );
             this.get("notify").alert(`${translated_attribute} ${message}`, {
-              closeAfter: 8000
+              closeAfter: config.environment === "test" ? null : 8000
```

Comment:
We should probably do this globally as the notifications are seemingly broken in all tests?

Reply:
https://github.com/puzzle/skills/issues/512